temp/userStories.adoc

```diff
@@ -0,0 +1,83 @@
+= Anforderungen allgemein
+
+[Note]
+====
+* Administratoren legen beliebig viele Diskussionsforen an
+* Gruppen sollen eigene Foren erstellen und verwalten können
+** diese dürfen nur von Mitgliedern der Gruppe lesbar sein (außer Admins)
+* es gibt anonyme Posts, sonst werden Klarnamen verwendet
+* Foren sollen einen in einen Moderationsmodus versetzt werden können
+* es soll Möglichkeiten geben differenziert in gesamten Foren und Teilen dessen zu suchen
+====
+
+== User Stories
+
+=== Student, Standard
+
+1. Student meldet sich in MOPS an.
+2. Student klickt auf ein `berechtigtes` Forum.
+3. Der Student `erstellt` einen Beitrag in einem der Topics.
+4. Der Student `bearbeitet` seinen Beitrag, um weiter Informationen einzupflegen.
+5. Der Student meldet sich später wieder an, um die Antworten zu `lesen`.
+
+=== Student, Suche/anonym
+
+1. Student `sucht` nach einem alten Beitrag im `gesamten` Forum.
+2. Er findet den gesuchten Beitrag.
+3. Er möchte eine ihm unangenehme Frage zum Beitrag `anonym` im Forum stellen.
```

Comment:
'Beitrag' --> 'Thread'

Reply:
To be decided: Can a user be 'anonym' whenever he/she wants to ? Or only in specific Foren ?